soc/arm/st_stm32/stm32l4/power.c

```diff
@@ -17,12 +17,10 @@
 LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);
 
 /* Invoke Low Power/System Off specific Tasks */
-void pm_power_state_set(enum power_states state)
+void pm_power_state_set(enum pm_state state)
 {
 	switch (state) {
-#ifdef CONFIG_PM_SLEEP_STATES
-#ifdef CONFIG_HAS_POWER_STATE_SLEEP_1
-	case POWER_STATE_SLEEP_1:
+	case PM_STATE_RUNTIME_IDLE:
```

Comment:
PM_STATE_SUSPEND_TO_IDLE according to https://github.com/zephyrproject-rtos/zephyr/issues/31162#issuecomment-756818940
Not sure to see the interest of using a new nomenclature if this is to use misleading names

Reply:
done